test/release/examples/programs/linkedList.chpl

```diff
@@ -36,7 +36,61 @@ class List {
   proc insert(value: eltType) {
     head = new Node(eltType, value, head);
   }
+ 
+  
+ //  
+ // deletes the first element from the list 
+ //
+
+ proc delete_first() {
+    if head == nil then {
+      writeln("list is empty");
```

Comment:
This writeln (and the following two) should probably be deleted. This is an example program, but it wouldn't generally be OK to put such writeln calls in a library routine.